src/Command/SyncThumbsCommand.php

```diff
@@ -160,6 +161,7 @@ public function execute(InputInterface $input, OutputInterface $output): int
             }
 
             //clear entity manager for saving memory
+            \assert($this->mediaManager instanceof BaseManager);
```

Comment:
This is correct or better directly type BaseManager instead of ManagerInterface (the manager interface does not have the getObjectManager method

Reply:
This is not correct.

Since the constructor use MediaManagerInterface, I could implement MediaManagerInterface without extending the BaseManager. This is a problem I had with multiple class from the AdminBundle:
An abstract class which implement an interface should not have more public method than the interface.

I solve this in SonataAdmin by deprecating a loooot of method in abstract classes, or adding some in the interface.

Here it worst than my case because it's in another bundle, so I would 
- Add an issue to the doctrine-extensions. In next-major either the method should not be public anymore, either it should be added to interface.
- Change the type in the construct to BaseManager or stop using this method. If you cannot stop using it means we have to add it to the interface.

Another way, BC, would be to create a new interface in doctrine-extensions which extends the ManagerInterface and add this method. The BaseManager will implements this new interface and you'll be able to use it in the constructor.

Reply:
Looking at the way you use this method, I would say it would be nice to not add getObjectManager to the interface.

Maybe you could add some `clear` method instead.
Or a ClearableManagerInterface could be created and here you would do
```
if ($this->mediaManager instanceof ClearableManagerInterface) {
     $this->mediaManager->clear();
}
```
This way it would work for both clearable and not clearable managers.

It would be interested to see if some others sonata projects use directly the `getObjectManager` method.

Reply:
Done